tools/build_etcrelease

```diff
@@ -30,7 +30,7 @@ ber_wsroot=$(cd "$(dirname "$0")/.." && pwd)
 ber_first=true
 
 ber_template="                     SmartOS %%VERSION%% x86_64
-                       Copyright %%YEAR%% MNX Cloud, Inc.
+                    Copyright %%YEAR%% Edgecast Cloud LLC.
```

Comment:
Just a small nit: shouldn't the copyright be Edgecast Cloud LLC (edgecast.io)?

Reply:
Not here.